tests/behat/bootstrap/TideCommonTrait.php

```diff
@@ -0,0 +1,130 @@
+<?php
```

Comment:
This test trait could be in Tide Core so that other tide modules could use it for paragraph tests.

Reply:
Not a bad idea :)